### PR TITLE
separate out privateKeyJwk

### PR DESCRIPTION
**Pull request by kdenhartog**
Related perma-id PR can be found here: https://github.com/perma-id/w3id.org/pull/2547

### CODE REVIEW

**Comment by kdenhartog** [comment]
@OR13 I noticed that there's a few examples in the spec that use the `privateKeyJwk` term, but don't utilize the context. Since this would modify the context that should be used for that term in a non-obvious way do you have a preference about how the text should be modified for the spec? I intend to have that text added as well, but wasn't sure what yours (or the other code owners) preference would be for this.

**Review by OR13 — CHANGES_REQUESTED** [review]
I don't see creating a new context file for a vocabulary item that is fundamental to producing a digital signature to be helpful.

I would welcome additional spec text however.

**Comment by kdenhartog** [comment]
I guess I don't quite agree with your ontological categorization, but that doesn't make it incorrect since it's subjective. My take is that this is more about separating private data from public data from an ontological perspective, but I understand where you're coming from as well. I'll go ahead and close this PR for now and leave it to you to consider if others should consider a different perspective in the VCWG.

**Pull request closed without merging**
